styles/css/vtex.product-price.css

```diff
@@ -0,0 +1,44 @@
+.listPrice {
+  color: #979899;
+  margin-bottom: .25rem;
+}
+
+.sellingPrice {
+  color: #3f3f40;
+  font-size: 1.25rem;
+}
+
+.sellingPriceValue {
+  font-size: 2.25rem;
+  font-weight: 700;
+}
+
+.installments {
+  color: #3f3f40;
+}
+
+.savings {
+  font-weight: 500;
+  color: #79B03A;
+}
+
+.sellingPriceValue--summary {
+  font-size: 1.2rem;
+  font-weight: 600;
+  color: #2E2E2E;
+}
+
+.savings--summary {
+  background: #8BC34A;
+  border-radius: 1000px;
```

Comment:
Why `1000px`?

Reply:
This was the value on Figma:  https://www.figma.com/file/DnpQxbX4N5CsYtr0xdiRRn/02.-Store---Price?node-id=4%3A0

Reply:
You can accomplish the same result using `border-radius: 50%` or `border-radius: 100%` (because this component will never be really big), but doesn't make a difference I think to change the value, so it could be this way